src/renderer/ducks/validator/actions.ts

```diff
@@ -83,14 +83,15 @@ export type SignedNewAttestation = (
     block: string,
     index: number,
     slot: number,
+    bitsIndex: number,
 ) => {
-    payload: {block: string; index: number; slot: number};
+    payload: {block: string; index: number; slot: number; bitsIndex: number};
```

Comment:
```suggestion
    payload: {block: string; index: number; slot: number; validatorIndexInCommittee: number};
```

Reply:
you should maybe throw error if this is < 0 (Don't think it can happen but to be on the safe side)

Reply:
how we should handle this error if happens?

Reply:
i can add case if is index `-1` (not found) to not call saga